### PR TITLE
[CSL-800] Add Quizzes module + tests

### PR DESCRIPTION
**Pull request by stanlp1**
Old branch got messy, so moving over to a new branch.

### CODE REVIEW

**Review by jjl014 — APPROVED** [review]
Nice, this looks great to me! Thanks for making all the changes ❤️ 

Would love to get a final sign-off from @sblaurock before merging/deploying.

**Comment by sblaurock** [comment]
Screen Shot 2022-10-19 at 7 05 54 PM

**Comment by sblaurock** [comment]
✅ 446 passing (1m)

**Pull request merged**
